pkg/program/shell/run_dir_unix_test.go

```diff
@@ -0,0 +1,102 @@
+// +build !windows,!plan9
+
+package shell
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/elves/elvish/pkg/util"
+)
+
+// RunDirPathTest is a test case to be used in getRunDirPath.
```

Comment:
It is customary to use an anonymous struct (if possible) with unexported fields in test data tables.